jobs/webcompat-kb/sql/webcompat_knowledge_base/views/scored_site_reports/view.sql

```diff
@@ -0,0 +1,105 @@
+  /* Individual score components for each bug.
+
+  These should always match the logic in `{{ project }}.{{ dataset }}.WEBCOMPAT_METRIC_SCORE_NO_SITE_RANK`
+  such that multiplying all the columns except severity score is equivalent to running that function.*/ scores AS (
+  SELECT
+    number,
+    SUM(
+    IF
+      (weights.lookup_type = "severity"
+        AND weights.lookup_value = CAST(site_reports.severity AS string), weights.score, 0)) AS severity_score,
+    SUM(
+    IF
+      (weights.lookup_type = "impact"
+        AND weights.lookup_value = JSON_VALUE(site_reports.user_story, "$.impact"), weights.score, 0)) AS impact_score,
+    SUM(
+    IF
+      (weights.lookup_type = "platform"
+        AND weights.lookup_value IN UNNEST(SPLIT(JSON_VALUE(site_reports.user_story, "$.platform"))), weights.score, 0)) AS platform_score,
+    SUM(
+    IF
+      (weights.lookup_type = "configuration"
+        AND weights.lookup_value = IFNULL(JSON_VALUE(site_reports.user_story, "$.configuration"), "general"), weights.score, 0)) AS configuration_score,
+    SUM(
+    IF
+      (weights.lookup_type = "users_affected"
+        AND weights.lookup_value = IFNULL(JSON_VALUE(site_reports.user_story, "$.affects"), "all"), weights.score, 0)) AS affects_score,
+    SUM(
+    IF
+      (weights.lookup_type = "patch_applied"
+        AND weights.lookup_value =
+      IF
+        ("webcompat:sitepatch-applied" IN UNNEST(site_reports.keywords),
+        IF
+          ("webcompat:platform-bug" IN UNNEST(site_reports.keywords), "platform-bug", "site-bug"), "none"), weights.score, 0)) AS intervention_score,
+    SUM(
+    IF
+      (weights.lookup_type = "branch"
+        AND weights.lookup_value = IFNULL(JSON_VALUE(site_reports.user_story, "$.branch"), "release"), weights.score, 0)) AS branch_score,
+  FROM
+    `{{ project }}.{{ dataset }}.site_reports` AS site_reports
+  CROSS JOIN
+    `{{ project }}.{{ dataset }}.dim_bug_score` AS weights
+  GROUP BY
+    number),
+  /* Computed scores for each bug
+
+  These could be inlined, but it's slightly easier to read if they're computed in one place
+  */ computed_scores AS (
+  SELECT
+    number,
+    `{{ ref('WEBCOMPAT_METRIC_SCORE_NO_SITE_RANK') }}`(keywords,
+      user_story) AS triage_score_no_rank,
+    `{{ ref('WEBCOMPAT_METRIC_SCORE_SITE_RANK_MODIFER') }}`(url,
+      `{{ ref('WEBCOMPAT_METRIC_YYYYMM') }}`()) AS site_rank_score
+  FROM
+    `{{ project }}.{{ dataset }}.site_reports` AS site_reports),
+  site_report_scores AS (
+  SELECT
+    site_reports.*,
+    severity_score,
+    impact_score,
+    platform_score,
+    configuration_score,
+    affects_score,
+    intervention_score,
+    site_rank_score,
+    branch_score,
+    triage_score_no_rank * site_rank_score AS triage_score,
+    IFNULL(host_categories.is_global_1000, FALSE) AS is_global_1000,
+    IFNULL(host_categories.is_sightline, FALSE) AS is_sightline,
+    IFNULL(host_categories.is_japan_1000, FALSE) AS is_japan_1000,
+  FROM
```

Comment:
looks like `is_japan_1000_mobile` is missing, production view has:
`IFNULL(host_categories.is_japan_1000, FALSE) AND is_mobile AS is_japan_1000_mobile,`